lib/Functions.rb

```diff
@@ -0,0 +1,188 @@
+class Functions
+    $debug_on = false
+    def weapon_confimation(weapon_p1, weapon_p2)
+        if $debug_on
+            puts "test1"
```

Comment:
`no “puts` and `get.chomp`, only in `bin/main.rb`

Reply:
Why should i remove this???
Its debug mode just for testing purpose, if im the future i get any bug i can debug more easily